herd/monad.mli

```diff
@@ -239,6 +239,18 @@ module type S =
         (unit t -> unit t) ->
           (A.location * A.V.v) list -> A.size_env -> unit code
 
+      (* Generate a code monad for initial instructions *)
+      val initinstructions :
+        (int -> Label.Set.t) ->
+          (A.proc * ((int * A.instruction) list)) list -> unit code
+
+      (* Generate a code monad for initial instructions *)
+      val init_writes_and_instr :
+        (unit t -> unit t) ->
+            (A.location * A.V.v) list -> A.size_env ->
+        (int -> Label.Set.t) ->
+          (A.proc * ((int * A.instruction) list)) list -> unit code
```

Comment:
Comments of these two functions are identical, this is puzzling.

Reply:
Changed the comment; init_writes_and_instr really combines initinstructions with initwrites. I simply haven't found an existing way of properly combining two "unit code" values.